tests/test_core.py

```diff
@@ -5406,6 +5406,7 @@ def test_fs_errorstack(self, js_engines=[NODE_JS]):
   @also_with_noderawfs
   def test_fs_llseek(self, js_engines=None):
     self.set_setting('FORCE_FILESYSTEM', 1)
+    self.emcc_args += ['-lnodefs.js']
```

Comment:
should `@also_with_noderawfs` do this? that would fix other tests too maybe.

Reply:
reverted